src/app/domains/domains.service.spec.ts

```diff
@@ -38,14 +38,45 @@ describe('DomainsService', () => {
 
     httpClientSpy.get.and.returnValue(of(domainResponse))
 
-    service.getData().subscribe(response => {
+    service.getData({ pageSize: 25, startsFrom: 0, count: 'true' }).pipe().subscribe(response => {
       expect(response).toEqual(domainResponse)
-      done()
     })
     expect(httpClientSpy.get.calls.count()).toEqual(1, 'one call')
   })
 
-  it('should return a single domain object detail when a single record is requested', (done: DoneFn) => {
+  it('should return array of domains when get all domains called ', () => {
+    const domainResponse = {
+      data: [{
```

Comment:
the domain response can be re-used for each test, avoid copy/pasting for each function

Reply:
done